hathor/cli/run_node.py

```diff
@@ -216,7 +225,11 @@ def prepare(self, *, register_resources: bool = True) -> None:
             status_server = resources_builder.build()
             if self._args.status:
                 assert status_server is not None
-                self.reactor.listenTCP(self._args.status, status_server)
+
+                if self._args.x_enable_ipv6:
+                    self.reactor.listenTCP(self._args.status, status_server, interface='::0')
+                else:
+                    self.reactor.listenTCP(self._args.status, status_server)
```

Comment:
Same here. Shouldn't we listen in both ipv4 and ipv6 interfaces?

Reply:
Answered in the other thread.